server/api/tumdev/campus_backend.proto

```diff
@@ -538,29 +537,35 @@ message Movie {
   string link = 16;
 }
 
-message SendFeedbackReply {}
 message SendFeedbackRequest {
-  string topic = 1;
-  string email = 2;
-  string email_id = 3;
-  string message = 4;
-  int32 image_count = 5;
-  double latitude = 6;
-  double longitude = 7;
-  string os_version = 8;
-  string app_version = 9;
-}
-
-message SendFeedbackImageReply {
-  string status = 1;
-}
+  message Metadata {
+    double latitude = 1;
+    double longitude = 2;
+    string osVersion = 3;
+    string appVersion = 4;
+  }
 
-message SendFeedbackImageRequest {
-  int32 id = 1;
-  int32 image_nr = 2;
-  //todo where does the file come from?
+  enum Recipient {
+    // Feedback for the TUM Dev Team
+    TUM_DEV = 0;
+    // Feedback for the general TUM Contact Form
+    TUM_CONTACT = 1;
+  }
+
+  // who is the feedback for
+  Recipient recipient = 1;
+  // the email address of the user
+  string from_email = 2;
+  // The actual message
+  string message = 3;
+  // Optional metadata which the user can choose to attach or not
+  Metadata metadata = 4;
```

Comment:
Is it worth separating this entity? If so, why?

Reply:
The user can choose to either include or not include this metadata for data protection reasons
=> grouping them on the API level as well seems like a sane choice to me (we can change this if you'd prefer)

Reply:
Yeah, I don't think modeling the request after what is optional/required is helping here. A comment for the field would suffice.